pyproject.toml

```diff
@@ -6,3 +6,7 @@ build-backend = "setuptools.build_meta"
 line-length = 120
 
 [tool.setuptools_scm]
+
+[tool.pytest.ini_options]
```

Comment:
was ist der unterschied zwischen `tool:pytest` (wie's vorher war) und `tool.pytest`?

Reply:
Die Syntax in `.toml` Dateien ist eine andere als in `.cfg` Dateien (wobei es nicht wirklich eine eigene Syntax für `cfg` gibt, diese ist meist gleich der `.ini` Syntax).
Die [Doku von pytest](https://docs.pytest.org/en/stable/customize.html) erklärt wo und wie man pytest definieren kann.
Generell ist gerade Einiges los was config-files in der Python-Welt angeht.
Zwar sagt [PEP518](https://www.python.org/dev/peps/pep-0518/) dass man es nur für package kram verwenden soll, aber die community fand den Gedanken cool, **eine** Datei für alle Settings zu verwenden. Daher wird dieses PEP wohl auch noch angepasst werden ^^
Auf der PEP Seite findest du auch eine [schöne Tabelle](https://www.python.org/dev/peps/pep-0518/#overview-of-file-formats-considered), die einen guten Überblick der möglichen Dateitypen für configs zeigt.